frontend/src/pages/Graph/GraphPage.tsx

```diff
@@ -472,7 +479,7 @@ class GraphPageComponent extends React.Component<GraphPageProps, GraphPageState>
             <ErrorBoundary
               ref={this.errorBoundaryRef}
               onError={this.notifyError}
-              fallBackComponent={<GraphErrorBoundaryFallback />}
+              fallBackComponent={GraphErrorBoundaryFallback()}
```

Comment:
Oh you were trying to return a component... sorry I did not want you to change a component by a method (although both thing works). In case you want to create a component from function, the return type is `React.ReactElement`, not `React.ReactNode` (although I think this changes in React 18)

Reply:
updated